Gopkg.toml

```diff
@@ -15,49 +15,41 @@ required = [ "k8s.io/code-generator/cmd/client-gen", "k8s.io/gengo/types", "gith
   non-go = false
   unused-packages = false
 
-[[prune.project]]
-  name = "github.com/openshift/library-go"
-  non-go = false
-  unused-packages = false
-
 [[constraint]]
   name = "github.com/google/uuid"
   revision = "8c31c18f31ede9fc8eae72290a7e7a8064e9b3e3"
 
 [[constraint]]
   name = "k8s.io/api"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.16.0"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.16.0"
 
-[[override]]
+[[constraint]]
   name = "k8s.io/apiextensions-apiserver"
-  # use origin fork with nullable support.
-  # TODO(sttts, sig-master): switch back to upstream version in 1.14
-  branch = "origin-4.1-kubernetes-1.13.4"
-  source = "https://github.com/openshift/kubernetes-apiextensions-apiserver"
+  version = "kubernetes-1.16.0"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.4"
+  version = "kubernetes-1.16.0"
```

Comment:
Don't you want to pin this to latest k8s which is 1.16.2?

Reply:
Took the one from library-go. Otherwise, dep is unhappy.